src/internationalization/dictionaries/pt.json

```
@@ -48,6 +48,21 @@
     "projects": "Projetos"
   },
   "about": {
+    "sections": {
+      "cesium": {
+        "title": "O que é o CeSIUM",
+        "subtitle": "O CeSIUM é um Centro de estudantes de Engenharia Informática da Universidade do Minho composto por alunos voluntários que têm como objectivo principal representar e promover o curso. É um núcleo unido e dinâmico capaz de proporcionar experiências únicas e enriquecedoras para a tua futura vida profissional.",
+        "description": "Procuramos organizar actividades de contéudo semelhante e complementar ao curso, assim como promover e divulgar acções de lazer e socialização entre estudantes e ex-alunos. Representamos ainda os interesses colectivos dos alunos de Engenharia Informática junto dos orgãos ou instituições para que a sua carreira académica seja mais rica."
+      },
+      "team": {
+        "title": "Team",
+        "subtitle": "O CeSIUM não existiria sem a nossa equipa incrível, que trabalha todos os dias para manter o núcleo a funcionar e tornar os anos dos estudantes no curso únicos. Conhece-a aqui."
+      },
+      "departments": {
+        "title": "Departamentos",
+        "short_description": "Funcionamos com 5 departamentos que se focam nas diferentes áreas da nossa atividade enquanto núcleo. Conhece-os aqui."
```

`subtitle` is a cleaner name 💪🏼 (also change this in the english dict)

```suggestion
        "subtitle": "Funcionamos com 5 departamentos que se focam nas diferentes áreas da nossa atividade enquanto núcleo. Conhece-os aqui."
```